app/src/main/java/org/simple/clinic/searchresultsview/SearchResultsItemType.kt

```diff
@@ -8,7 +8,7 @@ import org.simple.clinic.facility.Facility
 import org.simple.clinic.patient.PatientSearchResult
 import org.simple.clinic.summary.GroupieItemWithUiEvents
 import org.simple.clinic.widgets.PatientSearchResultItemView
-import org.simple.clinic.widgets.PatientSearchResultItemView.*
+import org.simple.clinic.widgets.PatientSearchResultItemView.PatientSearchResultViewModel
```

Comment:
A commit with the exact same message exists in `master` (1823ef2b3e651e8f536cb5583dc365fd29617725). Is this a rebase artifact?